filters/include/pcl/filters/bilateral.h

```diff
@@ -120,8 +112,20 @@ namespace pcl
         */
       inline void
       setSearchMethod (const KdTreePtr &tree)
-      { tree_ = tree; }
+      { tree_ = tree; }  
```

Comment:
You added spaces at the end of this line. Please undo that.
You also added quite many empty lines before and after the moved function. One empty line for separation is enough. Sorry for being so pedantic about this, but the code style should stay consistent.

Reply:
yep got it